docs/references/api/resource_embedding.rst

```diff
@@ -951,6 +953,13 @@ Both ``is.null`` and ``not.is.null`` can be included inside the `or` operator. F
 
   curl "http://localhost:3000/films?select=title,actors(*),directors(*)&or=(actors.is.null,directors.is.null)"
 
+You can also use ``not.is.null`` as a workaround for an ``or`` filter across multiple resources.
+For instance, to show the films whose actors **or** directors are named John:
```

Comment:
@laurenceisla We need a subtitle here, like "or across embedded resources". Also, "workaround" should be removed. Supporting `or` like this is intentional.. although not succinct.